simtools/model/model_parameters.py

```diff
@@ -42,22 +42,6 @@
     "parabolic_dish": {"type": int, "names": []},
 }
 
-# The coordinate system is aligned with Alt (x) and Az(y), so need to rotate the camera.
-# The angle depends on what coordinate system was provided by the instrument team.
-# Specifically the case of LST and NectarCam is a bit weird at 270 - 2*10.893,
-# where the factor two is because we need to undo the rotation in the code
-# and then actually rotate in the right direction.
-CAMERA_ROTATE_ANGLE = {
-    "LST": 248.214,
-    "FlashCam": 270,
-    "NectarCam": 248.214,
-    "SCT": 90,
-    "ASTRI": 90,
-    "1M": 270,
-    "GCT": 90,
-    "SST": 90,
-}
-
 CAMERA_RADIUS_CURV = {
```

Comment:
I guess this will also go at some point (independent of this PR)

Reply:
Yes! Kept this file open to fix this next (I don't think it's even used anywhere).